comp/logs/agent/agentimpl/agent_test.go

```diff
@@ -104,6 +104,8 @@ func (suite *AgentTestSuite) SetupTest() {
 	suite.configOverrides["logs_config.run_path"] = suite.testDir
 	// Shorter grace period for tests.
 	suite.configOverrides["logs_config.stop_grace_period"] = 1
+	// Set a short scan period to allow it to run in the time period of the tcp and http tests
+	suite.configOverrides["logs_config.file_scan_period"] = 1
```

Comment:
I wonder why this wasn't needed before

Reply:
Honestly not sure either 🤔 I might change it back if it doesn't affect anything.